File: output/CodeLlama/avatar/Java/Python/atcoder_ABC125_A.py
import sys

sec = int(sys.stdin.readline())
per_num = int(sys.stdin.readline())
max_sec = float(sys.stdin.readline()) + 0.5
ans_num = 0
for i in range(sec, max_sec, sec):
    ans_num += per_num
print(ans_num)

File: output/ChatGPT/avatar/Java/C++/atcoder_ABC125_A.cpp
#include <iostream>
using namespace std;

int main() {
    int sec, per_num;
    cin >> sec >> per_num;
    double max_sec;
    cin >> max_sec;
    max_sec += 0.5;
    int ans_num = 0;
    for (int i = sec; i < max_sec; i += sec) {
        ans_num += per_num;
    }
    cout << ans_num << endl;
}